transport_nantes/stripe_app/views.py

```diff
@@ -298,7 +298,8 @@ def stripe_webhook(request):
         logger.info("Stripe payment webhook succeeded.")
         logger.debug("Details attached to event : \n\n", "="*30, "\n", event)
         try:
-            make_donation_from_webhook(event)
+            donation = make_donation_from_webhook(event)
+            donation.save()
         except Exception as error_message:
             logger.debug("="*80, "\n", "Error while creating \
```

Comment:
Why are you sure this exception doesn't deserve to be visible?

Reply:
What do you mean by Visible ? Moving it up to Info ?

Reply:
Right, if that exception occurs, do you want anyone to be able to know?  Or is this only for debugging?

I'd say that if you get an exception processing a web hook, you'd want that to show in the log so that when the actual symptom arises (customer complaint, etc.) you can see something about what has happened.

You should read logs regularly to understand what your software is doing.  If yoiu can't tell from the log, it means you're not logging well.

Reply:
You're completely right, I've been just stuck in the mindset that debug = most complete log, so I definitely wanted to log the error if it occurred. 
That was without considering that we actually only check the info log, by default.